applications/teletype/src/teletype_util.erl

```diff
@@ -308,29 +313,24 @@ account_params(DataJObj) ->
 
 -spec find_account_params(wh_json:object(), ne_binary()) -> wh_proplist().
 find_account_params(DataJObj, AccountId) ->
-    case teletype_util:open_doc(<<"account">>, AccountId, DataJObj) of
-        {'ok', AccountJObj} -> build_account_params(AccountJObj);
+    case ?MODULE:open_doc(<<"account">>, AccountId, DataJObj) of
```

Comment:
you changed teletype_util:open_doc to ?MODULE:open_doc , what does this solve ?
why not change to open_doc ? if we move open_doc outside this module we will get compilation error, which i believe is better then getting the errors when running.